gossip/apply_genesis.go

```diff
@@ -184,7 +184,7 @@ func (s *Store) applyEpoch1Genesis(blockProc BlockProc, g opera.Genesis) (err er
 	s.SetHistoryBlockEpochState(es.Epoch, bs, es)
 	s.SetEpochBlock(blockCtx.Idx+1, es.Epoch)
 	s.SetBlockEpochState(bs, es)
```

Comment:
Why we export `SetBlockEpochState`?  i just checked, it is used within `gossip` package only

Reply:
It's a part of the public interface - that's the only method to mutate `BlockEpochState`. We don't see most of gossip.Store method used elsewhere because only `gossip` processes consensus messages

Reply:
Yes I get it what this method does. Since it is used only within `gossip` package, I would make  `SetBlockEpochState` private.

Reply:
If a method is used only in one package, it isn't yet sufficient to make it private. Private methods indicate that they are not a part of the "clean" interface and should be called with caution, `SetBlockEpochState` seems to be a part of the "clean" interface

Reply:
@rus-alex what do you think about it?

Reply:
`gossip.Store` is enough independent and could be used as storage anywhere, not only by gossip package. `SetBlockEpochState()` should be public as part of clean interface and as pair method with public `GetBlockEpochState()`.